Avoid calling out to git during dump - hangs on ppc64le.

--- a/lisp/loadup.el	2020-07-27 23:21:49.000000000 +0200
+++ b/lisp/loadup.el	2020-09-16 21:50:13.919715162 +0200
@@ -465,8 +465,8 @@
                                (string-to-number
                                 (substring name (length base) exelen)))
 			     files)))
-      (setq emacs-repository-version (ignore-errors (emacs-repository-get-version))
-            emacs-repository-branch (ignore-errors (emacs-repository-get-branch)))
+      ;; (setq emacs-repository-version (ignore-errors (emacs-repository-get-version))
+      ;;       emacs-repository-branch (ignore-errors (emacs-repository-get-branch)))
       ;; A constant, so we shouldn't change it with `setq'.
       (defconst emacs-build-number
 	(if versions (1+ (apply #'max versions)) 1))))
